packages/editor/src/lib/editor/Editor.ts

```diff
@@ -8021,6 +8037,21 @@ export class Editor extends EventEmitter<TLEventMap> {
 		return this
 	}
 
+	/**
+	 * Dispatch a focus event.
+	 *
+	 * @example
+	 * ```ts
+	 * editor.focus()
+	 * ```
+	 *
+	 * @public
+	 */
+	focus(): this {
+		this.focusManager.focus()
+		return this
+	}
```

Comment:
I wonder whether this should set the `instanceState.isFocused` to true if not already?

Reply:
or really maybe it should be the job of the focus manager

Reply:
i think possibly: that it's tricky. if there are multiple editors on the page, i could see an argument that you might want to temporarily translate/move a shape on one editor (Editor A), but the focus remains on a separate editor (Editor B). and that just because you focus Editor A, it doesn't mean that you want Editor A to take over control from Editor B. but perhaps that's overthinking it?